websites/V/Voiranime/presence.ts

```diff
@@ -23,79 +37,69 @@ presence.on(
 
-	if (path.includes("/liste-danimes")) {
-		presenceData.details = "Visite la page :";
-		presenceData.state = "Listes d'animes";
-	} else if (path.includes("/nouveaux-ajouts")) {
-		presenceData.details = "Visite la page :";
-		presenceData.state = "Nouveaux animes";
-	} else if (path.includes("/prochainement")) {
-		presenceData.details = "Visite la page :";
-		presenceData.state = "Prochains animes";
-	} else if (document.location.search.startsWith("?s")) {
-		presenceData.details = "Recherche un anime :";
-		presenceData.state = new URLSearchParams(document.location.search).get("s");
-		presenceData.smallImageKey = "search";
-	} else if (path.includes("/anime/")) {
-		const title = document.querySelector(
-			"#manga-reading-nav-head > div > div.entry-header_wrap > div > div.c-breadcrumb > ol > li:nth-child(2) > a"
-		)?.textContent;
-
-		presenceData.details = "Visite la page de l'anime :";
-		presenceData.state = document.querySelector(
-			"body > div.wrap > div > div.site-content > div > div.profile-manga > div > div > div > div.post-title > h1"
-		)?.textContent;
-		if (!isNaN(video.duration) && title) {
-			const [startTimestamp, endTimestamp] = presence.getTimestamps(
+			presenceData.details = "Visite la page de l'anime :";
+			presenceData.state = document.querySelector(
+				"div.post-title > h1"
+			)?.textContent;
+			if (!isNaN(video.duration) && title) {
+				const [startTimestamp, endTimestamp] = presence.getTimestamps(
 					video.currentTime,
 					video.duration
-				),
-				[, epAndSeason] = document
-					.querySelector(
-						"#manga-reading-nav-head > div > div.entry-header_wrap > div > div.c-breadcrumb > ol > li.active"
-					)
-					.textContent.split("-");
+				);
 
-			presenceData.details = title;
-			presenceData.state = epAndSeason;
-			presenceData.startTimestamp = startTimestamp;
-			presenceData.endTimestamp = endTimestamp;
-			presenceData.smallImageKey = video.paused ? "pause" : "play";
-			presenceData.smallImageText = video.paused
-				? (await strings).pause
-				: (await strings).play;
-			presenceData.buttons = [
-				{
-					label: "Regarder l'épisode",
-					url: document.location.href,
-				},
-				{
-					label: "Voir l'anime",
-					url: document
-						.querySelector(
-							"#manga-reading-nav-head > div > div.entry-header_wrap > div > div.c-breadcrumb > ol > li:nth-child(2) > a"
-						)
-						.getAttribute("href"),
-				},
-			];
-			if (video.paused) {
-				delete presenceData.startTimestamp;
-				delete presenceData.endTimestamp;
+				presenceData.details = title.textContent;
+				presenceData.state = document
+					.querySelector("li.active")
+					.textContent.split("-")[1];
+				presenceData.startTimestamp = startTimestamp;
+				presenceData.endTimestamp = endTimestamp;
+				presenceData.smallImageKey = video.paused ? "pause" : "play";
+				presenceData.smallImageText = video.paused
+					? (await strings).pause
+					: (await strings).play;
```

Comment:
await the strings at the top of the file instead of where it is used.

Reply:
Okay, I'll fix that in the next days